source/jsonnet/england-wales/ccs/blocks/who-lives-here/anyone_else_temp_away_list_collector.jsonnet

```diff
@@ -52,7 +52,7 @@ local editQuestion(questionTitle) = {
   },
   remove_answer: {
     id: 'anyone-else-temp-away-remove-confirmation',
-    value: 'Yes, I want to remove this person',
+    value: 'Yes, remove this person',
```

Comment:
The CCS schemas aren't mentioned in the card or the spreadsheet of content changes. Should it be?

Reply:
Reverted the change in CCS, thanks 👍